### PR TITLE
Cover case with API Bundles in the Compass Runtime Agent tests

### PR DESCRIPTION
**Pull request by VOID404**
**Description**

Changes proposed in this pull request:

- Test creating services in Application CRD based on Compass API Bundles 

**Related issue(s)**
#13977

### CODE REVIEW

**Comment by VOID404** [comment]
image

Tests now skip, if dependencies fail

File: tests/components/application-connector/test/compass-runtime-agent/synchronisation_test.go [hunk 1]
```diff
@@ -44,13 +48,29 @@ func (cs *CompassRuntimeAgentSuite) TestApplication() {
...
+		}
+		_ = cs.updateAndWait(applicationInterface, synchronizedCompassAppName, applicationID)
 
 		err = cs.appComparator.Compare(cs.T(), updatedAppName, synchronizedCompassAppName)
 		cs.NoError(err)
+
+		correctState = err == nil
+	})
+
+	cs.Run("With bundles compare", func() {
```
Comment (akgalwas) [inline tests/components/application-connector/test/compass-runtime-agent/synchronisation_test.go:67]: Please consider adding a bundle to the app used for `Compass Runtime Agent should create Application` test. This approach reduces number of GraphQL calls.

**Review by akgalwas — APPROVED** [review]

**Pull request merged**